index.html

```diff
@@ -6216,6 +6237,9 @@ <h2>Changes since JSON-LD Community Group Final Report</h2>
     <li>The <a href="#iri-compaction">IRI compaction algorithm</a> may generate an error if the result is an
       <a>absolute IRI</a> which could be confused with a compact IRI in the
       <a>active context</a>.</li>
+    <li>By default, all contexts are propagated when traversing <a>node objects</a>, other than
+      type-scoped contexts. This can be controlled using the <code>@preserve</code>
```

Comment:
```suggestion
      type-scoped contexts. This can be controlled using the <code>@propagate</code>
```

Reply:
Perhaps an indication of difficulty, but I find myself often typing `@preserve` when I mean `@propogate`.